rdr_service/.configs/current_config.json

```diff
@@ -1,38 +1 @@
-  "biobank_id_prefix": ["Z"],
-  "biobank_samples_shards": [
-    "2"
-  ],
-  "user_info": {
-    "example@example.com": {
-      "roles": ["ptc", "healthpro", "exporter", "storefront", "rdr"],
-      "whitelisted_ip_ranges": {
-	"ip6": ["::1/64"],
-	"ip4": ["127.0.0.1/32"]
-      }
-    },
-    "awardee-pitt@pmi-drc-api-test.iam.gserviceaccount.com": {
-      "roles": ["awardee_sa"],
-      "awardee": "PITT",
-      "whitelisted_ip_ranges" : {
-	"ip4": ["136.142.28.15", "136.142.28.122", "136.142.28.123", "136.142.28.76", "136.142.202.70", "136.142.202.71", "136.142.202.72", "136.142.202.73", "136.142.202.74", "136.142.202.75", "136.142.202.76", "136.142.202.77", "136.142.202.78"]
-      }
-    }
-  },
-  "biobank_samples_bucket_name": [
-    "app_default_bucket"
-  ],
-  "ghost_id_bucket": [
-    "all-of-us-rdr-test-ghost-accounts"
-  ],
-  "consent_pdf_bucket": [
-    "app_default_bucket"
-  ],
-  "metrics_shards": [
-    "10"
-  ],
-  "mayolink_creds": ["rdr-mayolink-creds-test.json"],
-  "mayolink_endpoint": ["https://test.orders.mayocliniclabs.com/orders/create.xml"]
-}
+{"baseline_ppi_questionnaire_fields": ["questionnaireOnTheBasics", "questionnaireOnOverallHealth", "questionnaireOnLifestyle"], "ppi_questionnaire_fields": ["questionnaireOnTheBasics", "questionnaireOnOverallHealth", "questionnaireOnLifestyle", "questionnaireOnFamilyHealth", "questionnaireOnHealthcareAccess", "questionnaireOnMedicalHistory", "questionnaireOnMedications"], "baseline_sample_test_codes": ["1ED04", "1ED10", "1HEP4", "1PST8", "2PST8", "1SST8", "2SST8", "1PS08", "1SS08", "1UR10", "1CFD9", "1PXR2", "1UR90", "2ED10"], "dna_sample_test_codes": ["1ED10", "2ED10", "1ED04", "1SAL", "1SAL2"], "native_american_race_codes": ["WhatRaceEthnicity_AIAN"], "internal_status_email_sender": ["pmi-drc-alerts@googlegroups.com"], "internal_status_email_recipients": ["pmi-drc-alerts+nonprod@googlegroups.com"], "days_to_delete_keys": [3], "ehr_status_bigquery_view_participant": [], "ehr_status_bigquery_view_organization": [], "allow_nonprod_requests": true, "biobank_id_prefix": ["Z"], "biobank_samples_shards": ["2"], "user_info": {"example@example.com": {"roles": ["ptc", "healthpro", "exporter", "storefront", "rdr"], "whitelisted_ip_ranges": {"ip6": ["::1/64"], "ip4": ["127.0.0.1/32"]}}, "awardee-pitt@pmi-drc-api-test.iam.gserviceaccount.com": {"roles": ["awardee_sa"], "awardee": "PITT", "whitelisted_ip_ranges": {"ip4": ["136.142.28.15", "136.142.28.122", "136.142.28.123", "136.142.28.76", "136.142.202.70", "136.142.202.71", "136.142.202.72", "136.142.202.73", "136.142.202.74", "136.142.202.75", "136.142.202.76", "136.142.202.77", "136.142.202.78"]}}}, "biobank_samples_bucket_name": ["app_default_bucket"], "ghost_id_bucket": ["all-of-us-rdr-test-ghost-accounts"], "consent_pdf_bucket": ["app_default_bucket"], "metrics_shards": ["10"], "mayolink_creds": ["rdr-mayolink-creds-test.json"], "mayolink_endpoint": ["https://test.orders.mayocliniclabs.com/orders/create.xml"]}
```

Comment:
Did running tests replace this file ? If so, perhaps we should add it to gitignore.

Reply:
Added to gitignore.